### PR TITLE
45930/lightnet in readme

### PR DESCRIPTION
**Pull request by 45930**
Adding info in the readme about how to run the lightnet.

### CODE REVIEW

**Comment by shimkiv** [comment]
@45930 thanks for PR.
FYI we have another WIP PR that will provide documentation for the "lightnet": https://github.com/o1-labs/docs2/pull/780
Also o1js repo has almost the same docs: https://github.com/o1-labs/o1js/blob/main/README-dev.md#test-zkapps-against-lightnet-network
So i'm not sure if we need this PR to be merged.

**Comment by 45930** [comment]
You can close this PR, but I'd highly recommend documenting usage of the lightnet _in_ zkapp-cli, or linking to a more robust documentation hosted elsewhere.  Why would I look in the o1js dev readme to learn how to use the zkapp-cli?

**Comment by shimkiv** [comment]
> You can close this PR, but I'd highly recommend documenting usage of the lightnet _in_ zkapp-cli, or linking to a more robust documentation hosted elsewhere. Why would I look in the o1js dev readme to learn how to use the zkapp-cli?

Once mentioned documentation will be published the `zkapp-cli` also will be updated with links https://github.com/o1-labs/zkapp-cli/issues/519

**Review by mitschabaude — APPROVED** [review]
I think this is valuable either way! We also document `zk deploy` on the README, why not lightnet?

**Comment by shimkiv** [comment]
I don't mind at all but

> I think this is valuable either way! We also document `zk deploy` on the README, why not lightnet?

Because this PR adds some description about certain sub-commands and APIs. It does not describes what the lightnet is and what other sub-commands are used for. At the same time adds yet another item for maintenance in case of future changes.
I'd prefer this to also be reviewed by @barriebyron to keep in consistent.

**Comment by 45930** [comment]
@shimkiv I agree that Barrie's docs are phenomenal and the existing docs in o1js readme are better than these proposed changes.  I could edit this PR to just copy the o1js docs if you want, with the goal of replacing that section entirely with a link to the full docs after https://github.com/o1-labs/docs2/pull/780 is merged.

At the moment this feature is totally undocumented in a zkapp-dev-facing location (o1js doc is for o1 devs, not o1js consumers).  But we are still using it.  A search in discord will reveal many recommendations to "run `zk lightnet start`".  I only meant to add some context to the readme here, since I was unaware of the other documentation efforts.

**Comment by barriebyron** [comment]
> @shimkiv I agree that Barrie's docs are phenomenal and the existing docs in o1js readme are better than these proposed changes. I could edit this PR to just copy the o1js docs if you want, with the goal of replacing that section entirely with a link to the full docs after [o1-labs/docs2#780](https://github.com/o1-labs/docs2/pull/780) is merged.
> 
> At the moment this feature is totally undocumented in a zkapp-dev-facing location (o1js doc is for o1 devs, not o1js consumers). But we are still using it. A search in discord will reveal many recommendations to "run `zk lightnet start`". I only meant to add some context to the readme here, since I was unaware of the other documentation efforts.

@45930 Thanks for taking the incentive to help our devs discover and use Lightnet. We appreciate you!

I circled in with our product manager. The strategy is to have the docs serve as the canonical resource, so the most important thing here is the description and link to the doc.

We agree, @shimkiv , that we do not want to duplicate technical information that requires maintenance. Please can you use GitHub suggestions to remove the out-of-scope lines?

I've added a Lightnet description, the basic start command, the explorer command, and links to the Lightnet doc (see the preview at https://docs2-git-lightnet-minadocs.vercel.app/zkapps/testing-zkapps-lightnet#lightweight-mina-explorer live doc after PR is approved and merged).

**Comment by shimkiv** [comment]
Docs were published: https://docs.minaprotocol.com/zkapps/testing-zkapps-lightnet

**Review by barriebyron — APPROVED** [review]
thanks for this PR @45930

Now that Lightnet docs are published, let's put the main command and the explorer and then maintain the details in the doc

**Review by barriebyron — APPROVED** [review]
Great idea @45930  thanks Coby! I went ahead and made product-approved updates to your useful PR

File: README.md [hunk 1]
```diff
@@ -173,13 +173,46 @@ zk deploy <alias>
...
+
+Join us in these zkApps channels:
+
+* [#zkapps-developers](https://discord.com/channels/484437221055922177/915745847692636181) to meet other developers building zkApps with o1js
+* [#zkapps-general](https://discord.com/channels/484437221055922177/910549624413102100) to ask general questions about zkApps, how to use a zkApp, and so on
+* [#zkapps-questions](https://discord.com/channels/484437221055922177/1047214314349658172) to ask zkApps-related questions and see Q&A history
+
+## Feedback
+
+To provide feedback, first review existing issues. You can create issues for zkApp CLI and Lightnet in this [zkapp-cli rep](https://github.com/o1-labs/zkapp-cli/issues).
```
Comment (shimkiv) [inline README.md:214]:
```suggestion
To provide feedback, first review existing issues. You can add contributions, feature requests or issues to this GitHub repository.
```

File: README.md [hunk 2]
```diff
@@ -173,13 +173,46 @@ zk deploy <alias>
...
 Note: When you deploy to an alias for the first time, you are prompted to choose which smart contract you want to deploy from those that exist as _named_ exports in your project. The name of the smart contract that you choose is remembered by being saved into your `config.json` for this alias. For safety, the next time you run `zk deploy <alias>` this _same_ smart contract automatically deploys to this alias. See [Tutorial 3: Deploy to a Live Network](https://docs.minaprotocol.com/zkapps/tutorials/deploying-to-a-network).
 
+## Testing your zkApp
+
+To test your zkApps, you first create automated tests for your smart contract and test with a simulated local blockchain. See [Testing zkApps Locally](https://docs.minaprotocol.com/zkapps/testing-zkapps-locally).
+
+Before you test with a live network, use Lightnet to test your zkApp locally on an accurate representation of a Mina blockchain. 
+
+```sh
+zk lightnet
```
Comment (shimkiv) [inline README.md:189]:
```suggestion
zk lightnet --help
```

**Review by shimkiv — APPROVED** [review]
LGTM with couple of comments

File: README.md (hunk 2 above)
Comment (barriebyron) [reply README.md:189]: got it now!

Comment (barriebyron) [reply README.md:189]: sorry to miss this, I fixed in https://github.com/o1-labs/zkapp-cli/pull/572 

Thanks @shimkiv

**Pull request merged**